assets/js/app/Data.js

```diff
@@ -4,8 +4,60 @@
 
 "use strict";
 
+/* A named collection of data sources used to draw a plot*/
+LocusZoom.DataSources = function() {
```

Comment:
I've really only been treating "Data" as a namespace to try to not pollute the main LocusZoom namespace. There's not really a need for it and it should probably disappear. Rather than LocusZoom.Data.GeneSource. It should be LocusZoom.GeneDataSource or something.